### PR TITLE
Create Serverless CMS Cookbook for Vue V3

### PR DESCRIPTION
**Pull request by orlyohreally**
A cookbook recipe on how to quickly build a CMS-powered blog with Vue.js and ButterCMS

### CODE REVIEW

**Comment by jakelumetta** [comment]
@orlyohreally Images are broken in the .md file: https://github.com/ButterCMS/docs/blob/ae8471f8d47b7c3937abec66144349094b8920d5/src/cookbook/serverless-blog.md

Let's make sure they are up to date images as well in terms of our new blog post editor UI. Can grab pictures from: https://buttercms.com/features/ if needed

**Comment by orlyohreally** [comment]
> @orlyohreally Images are broken in the .md file: https://github.com/ButterCMS/docs/blob/ae8471f8d47b7c3937abec66144349094b8920d5/src/cookbook/serverless-blog.md
> 
> Let's make sure they are up to date images as well in terms of our new blog post editor UI. Can grab pictures from: https://buttercms.com/features/ if needed

I have replaced the image with post editor UI with the one from the features page. I have checked and it seems that other recipes also have images not displaying when viewing .md files, but they work when you run the project

File: src/cookbook/serverless-blog.md [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+            <h2>{{ post.title }}</h2>
+            <p>{{ post.summary }}</p>
+            </article>
+        </router-link>
+        </div>
+    </div>
+</template>
+
+<script>
+import { butter } from "@/buttercms"
```
Comment (courcelan) [inline src/cookbook/serverless-blog.md]: lets move this down below `import { RouterLink }` as we are third-party and want to ensure that Vue is the prime import.

File: src/cookbook/serverless-blog.md [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+        {{ post.meta.next_post.title }}
+    </router-link>
+</div>
+</template>
+
+<script>
+import { defineComponent, onMounted, ref, unref } from "vue"
+import { RouterLink, useRoute } from "vue-router"
+
+import { butter } from "@/buttercms.js"
```
Comment (courcelan) [inline src/cookbook/serverless-blog.md]: Lets be consistent here and ensure this is `import { butter } from "@/buttercms"`

File: src/cookbook/serverless-blog.md [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+        {{ post.meta.next_post.title }}
+    </router-link>
+</div>
+</template>
+
+<script>
+import { defineComponent, onMounted, ref, unref, watch } from "vue"
+import { RouterLink, useRoute } from "vue-router"
+
+import { butter } from "@/buttercms.js"
```
Comment (courcelan) [inline src/cookbook/serverless-blog.md]: Lets be consistent here and ensure this is import { butter } from "@/buttercms"

File: src/cookbook/serverless-blog.md [hunk 4]
```diff
@@ -0,0 +1,349 @@
...
+* [Authors](https://buttercms.com/docs/api/?javascript#authors)
+
+Here's an example of listing all categories and getting posts by category. Call these methods on the `onMounted` hook inside `setup`:
+
+```javascript
+onMounted( async () => {
+    const categories = await getCategories()
+    const postsByCategory = await getPostsByCategory( "example-category" )
+
+    console.log(categories, postsByCategory)
```
Comment (courcelan) [inline src/cookbook/serverless-blog.md]: Lets remove the `console.log` unless we are describing why thats in the code. if it should remain, then lets do:

`console.log({ categories, postsByCategory })`

which will output:

```
{
  categories: [ ... ],
  postsByCategory: [ ... ]
}
```

this is easier in the console to tell what data you are looking at.

**Review by courcelan — CHANGES_REQUESTED** [review]
Added some recommendations for stylistic updates.

Looks good otherwise!

**Review by courcelan — APPROVED** [review]
Looks good from here.

**Comment by jakelumetta** [comment]
Awesome thanks @courcelan

**Pull request merged**